pkg/admission/pluginconfig_webhook.go

```diff
@@ -68,6 +68,11 @@ func DefaultPluginConfig(ctx context.Context, c client.Client, obj runtime.Objec
 		return err
 	}
 	pluginConfig.Spec.OptionValues = optionValues
+
+	// Default the ReleaseNamespace to the organization namespace if not set.
```

Comment:
For follow-up PRs: We might also want to consider providing defaults in the Plugin. E.g. kube-system is a common home for ingress-nginx, cert-manager namespace usually hosts for digicert-issuer, cert-manager, etc.